backend/src/data/models/group/group.model.ts

```diff
@@ -1,14 +1,54 @@
+import { Model, RelationMappings } from 'objection';
 import { TableName } from '~/common/enums/enums';
 import { AbstractModel } from '../abstract/abstract.model';
+import {
+  Worker as WorkerModel,
+  Permission as PermissionModel,
+} from '~/data/models/models';
 import { GroupTableField } from './group-table-field.enum';
 
 class Group extends AbstractModel {
   [GroupTableField.NAME]: string;
   [GroupTableField.TENANT_ID]: string;
+  users: { id: string; name: string }[] | [];
+  permissions: { id: string; name: string }[];
```

Comment:
```suggestion
  [GroupTableField.USERS]: { id: string; name: string }[];
  [GroupTableField.PERMISSIONS]: { id: string; name: string }[];
```
also, create a type instead of object literals